src/utils/system.rs

```diff
@@ -190,14 +192,14 @@ pub fn get_model() -> Option<String> {
 }
 
 #[cfg(target_os="macos")]
-pub fn get_family() -> Option<String> {
+pub fn get_family(config: &Config) -> Option<String> {
```

Comment:
It would be great if the macOS implementation also tries to read the overrides from `config` and then falls back to the current implementation if not set. Same goes for `get_model`.